File: go/1019.go
package main

//Funcional no beecrowd!
import (
	"fmt"
	"math"
)

func main() {
	var segundos int
	fmt.Scan(&segundos)
	hora := int(math.Pow(60, 2))
	minuto := 60

	horas := int(segundos / hora)
	segundos -= horas * hora
	minutos := int(segundos / minuto)
	segundos -= minutos * minuto

	fmt.Printf("%d:%d:%d\n", horas, minutos, segundos)

}

File: Lua/Iniciante/1019.lua
-- Funcional no beecrowd
local A = io.read("n")
local h = A // 3600
A = A - h * 3600
local m = A // 60
A = A - m * 60
local s = A

print(string.format("%d:%d:%d", h, m, s))